pkgs/development/python-modules/alarmdecoder/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, buildPythonPackage
+, fetchFromGitHub
+, pyserial, pyftdi, pyusb, pyopenssl
+, nose, mock }:
+
+buildPythonPackage rec {
+  pname = "alarmdecoder";
+  version = "1.13.9";
+
+  src = fetchFromGitHub {
+    owner = "nutechsoftware";
+    repo = "alarmdecoder";
+    rev = version;
+    sha256 = "0plr2h1qn4ryawbaxf29cfna4wailghhaqy1jcm9kxq6q7b9xqqy";
+  };
+
+  buildInputs = [ pyserial pyftdi pyusb pyopenssl ];
+
+  doCheck = true;
```

Comment:
https://github.com/nutechsoftware/alarmdecoder/issues/45

Reply:
Otherwise you an also skip the tests on python3 using `nosetests --exclude`.